pandas/core/generic.py

```diff
@@ -3078,11 +3078,12 @@ def to_latex(
             centered labels (instead of top-aligned) across the contained
             rows, separating groups via clines. The default will be read
             from the pandas config module.
-        caption : str, optional
-            The LaTeX caption to be placed inside ``\caption{}`` in the output.
+        caption : str or tuple, optional
+            Tuple (full_caption, short_caption),
+            which results in \caption[short_caption]{full_caption};
+            if a single string is passed, no short caption will be set.
 
             .. versionadded:: 1.0.0
```

Comment:
```suggestion
            .. versionchanged:: 1.2.0
```

Reply:
But I think I should keep ``.. versionadded`` for the consistency. Shouldn't I?

Reply:
sure.

can have both. and add a one-liner describing the changes.

Reply:
Added these tags for caption, position (PR https://github.com/pandas-dev/pandas/pull/35284) and ``to_latex`` function itself.